legend-engine-server/pom.xml

```diff
@@ -485,6 +489,16 @@
             <artifactId>legend-pure-code-compiled-core-relational</artifactId>
             <scope>test</scope>
         </dependency>
+<!--        <dependency>-->
```

Comment:
why comment? remove it

Reply:
removed , was testing build job on github